doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -65,6 +65,9 @@ Building OpenCV from Source Using CMake
     For example
     @code{.bash}
     cmake -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=/usr/local ..
+@note
+    use cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local ..   ,without space if the above doesn't work
```

Comment:
- Please, wrap the command line with backticks <code>`</code> to make it look like other code.
- Also, add a space after comma `, `
- Replace `without space` -> `without spaces after -D`